.github/workflows/linux-eic-shell.yml

```diff
@@ -306,13 +317,22 @@ jobs:
       with:
         name: sim_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4hep.root
     - uses: cvmfs-contrib/github-action-cvmfs@v3
+    - name: Check dynamic library loader paths
+      uses: eic/run-cvmfs-osg-eic-shell@main
+      with:
+        platform-release: "jug_xl:nightly"
+        setup: /opt/detector/setup.sh
+        run: |
+          export LD_LIBRARY_PATH=$PWD/lib:$LD_LIBRARY_PATH
```

Comment:
`export LD_LIBRARY_PATH=${LD_LIBRARY_PATH//\/usr\/local\/lib}`

Reply:
1. We can't filter out `/usr/local/lib` entirely since it then doesn't find dd4hep and podio
2. We can't filter out with this pattern because it also mangles `/usr/local/lib/root` which is required for root
3. We can't construct LD_LIBRARY_PATH ourselves entirely because it needs the part in /opt/detector/epic-nightly/lib (which we can hardcode, but probably not ideal)

Reply:
I was trying to construct it from nothing, and ended up with
```
export LD_LIBRARY_PATH=$PWD/lib:/opt/detector/epic-nightly/lib:$(spack location -i podio)/lib:$(spack location -i dd4hep)/lib:/usr/local/lib/root
```

Reply:
That just looked more like a headache to maintain... even for a short time.

Reply:
Actually, what I had tried before is:
```
export LD_LIBRARY_PATH=$PWD/lib:/opt/detector/epic-${DETECTOR_VERSION}/lib:$(spack location -i podio)/lib:$(spack location -i dd4hep)/lib:$(spack location -i root)/lib/root
```
which works, but it just doesn't look that nice (especially not to copy it multiple times in the code, and we can't even use it in the `env:` section on account of the `$( )` calls).